packages/ui-components/src/components/select-multi-options/select-multi-options.tsx

```diff
@@ -193,6 +258,13 @@ export class KvSelectMultiOptions implements ISelectMultiOptionsConfig, ISelectM
 					</slot>
 				)}
 				{this.renderOptions()}
+				{this.shortcuts && (
+					<kv-select-shortcuts-label slot="select-footer">
```

Comment:
Should this in the future be a slot? I imagine that after this update, they will look at this like a custom thing.

Reply:
You're right, I'll add a slot here 👍